pyproject.toml

```diff
@@ -25,6 +26,7 @@ invoke = "^1.3.0"
 semantic-version = "^2.8.4"  # For sdsstools.releases
 docutils = "^0.16"
 six = "^1.14.0"
+packaging = "^20.4"
 
```

Comment:
Should `importlib_metadata` be added to the list of dependencies?

Reply:
I added it for Python <3.7. It must have been installed by some other package, or at least the tests for Python 3.6 in the workflow didn't fail, but it's now added explicitely.